pandapower/plotting/to_html.py

```diff
@@ -194,8 +194,9 @@ def _net_to_html(net, respect_switches=True, include_lines=True, include_trafos=
 
     tables = []
     if show_tables:
-        for name in ['bus', 'trafo', 'line', 'dcline', 'load', 'ext_grid', 'gen', 'sgen',
-                     'res_bus', 'res_trafo', 'res_line', 'res_dcline', 'res_load', 'res_ext_grid', 'res_gen', 'res_sgen']:
+        for name in ['bus', 'trafo', 'line', 'dcline', 'load', 'asymmetric_load', 'ext_grid', 'gen', 'sgen',
+                     'res_bus', 'res_trafo', 'res_line', 'res_dcline', 'res_load', 'res_ext_grid', 'res_gen', 'res_sgen',
+                     'res_bus_3ph', 'res_trafo_3ph', 'res_line_3ph', 'res_load_3ph', 'res_asymmetric_load_3ph', 'res_ext_grid_3ph', 'res_sgen_3ph']:
             item = getattr(net, name)
```

Comment:
Thanks for the contribution! What do you think about adding `if len(item): ... ` in Line 201 to avoid empty tables in the HTML output?

Reply:
Seems great. Added as additional commit.

Reply:
looks good, thanks!